src/cpp/transport/TCPTransportInterface.cpp

```diff
@@ -782,21 +780,30 @@ bool TCPTransportInterface::Receive(TCPChannelResource *pChannelResource, octet*
                 }
                 else
                 {
-                    //TCPHeader tcp_header;
-                    //memcpy(&tcp_header, header, TCPHeader::getSize());
                     size_t body_size = tcp_header.length - static_cast<uint32_t>(TCPHeader::getSize());
-                    //logInfo(RTCP_MSG_IN, " Received [TCPHeader] (CRC=" << tcp_header.crc << ")");
 
                     if (body_size > receiveBufferCapacity)
                     {
                         logError(RTCP_MSG_IN, "Size of incoming TCP message is bigger than buffer capacity: "
-                            << static_cast<uint32_t>(body_size) << " vs. " << receiveBufferCapacity << ".");
+                            << static_cast<uint32_t>(body_size) << " vs. " << receiveBufferCapacity << ". " <<
+                            "The full message will be drop.");
```

Comment:
```suggestion
                            "The full message will be dropped.");
```

Reply:
I'll change it along the other requested changes to avoid an additional commit.